packages/salesforcedx-vscode-core/src/commands/auth/orgLogout.ts

```diff
@@ -28,41 +28,41 @@ import {
 } from '../util';
 import { ScratchOrgLogoutParamsGatherer } from './authParamsGatherer';
 
-export class ForceAuthLogoutAll extends SfdxCommandletExecutor<{}> {
-  public static withoutShowingChannel(): ForceAuthLogoutAll {
-    const instance = new ForceAuthLogoutAll();
+export class OrgLogoutAll extends SfdxCommandletExecutor<{}> {
+  public static withoutShowingChannel(): OrgLogoutAll {
+    const instance = new OrgLogoutAll();
     instance.showChannelOutput = false;
     return instance;
   }
 
   public build(data: {}): Command {
     return new SfdxCommandBuilder()
-      .withDescription(nls.localize('force_auth_logout_all_text'))
-      .withArg('force:auth:logout')
+      .withDescription(nls.localize('org_logout_all_text'))
+      .withArg('org:logout')
```

Comment:
@CristiCanizales Is `force:auth:logout` deprecated? Because [here ](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_org_commands_unified.htm#cli_reference_org_logout_unified:~:text=Aliases%20for%20org%20logout)it says aliases for logout.

Reply:
https://github.com/forcedotcom/cli/blob/main/releasenotes/sfdx/README.md#71900-mar-2-2023
<img width="465" alt="image" src="https://github.com/forcedotcom/salesforcedx-vscode/assets/113132642/eaf6c724-634b-42aa-9a1f-125c57d77998">

Reply:
But we are getting the warnings! So 👍🏻